statsmodels/stats/moment_helpers.py

```diff
@@ -87,67 +109,88 @@ def mnc2cum(mnc):
 
     http://en.wikipedia.org/wiki/Cumulant#Cumulants_and_moments
     '''
-    mnc = [1] + list(mnc)
-    kappa = [1]
-    for nn,m in enumerate(mnc[1:]):
-        n = nn+1
-        kappa.append(m)
```

Comment:
Given that multidimensional is not supported, we can just use ndarrays in the multidimensional case.

However, we cannot just replace mnc by a copy, kappa needs to have the copy, mnc needs to be the original in the loop below.